draft-ietf-scitt-architecture.md

```diff
@@ -363,36 +366,41 @@ This section describes at a high level, the three main roles and associated proc
 
 ## Signed Statement Issuance and Registration
 
-### Issuer Identity
+### Structure and Content of Signed Statements
+
+Before an Issuer is able to produce Signed Statements, it must first create an identifier and obtain an identity document that is acceptable to the Transparency Service.
+Transparency Services MUST support the capability to associate an X.509v3 certificate with a Signed Statement using a hash (thumbprint) of the certificate as specified in {{Section 2 of RFC9360}} by supporting the `x5t` COSE header parameter. The `x5t` COSE header parameter MUST be included in the protected header of a Signed Statement's COSE envelope. The mechanisms how Transparency Services obtain corresponding X.509v3 certificates, e.g., as part of enforcing Registration Policy, is out-of-scope of this document.
+Transparency Services MAY support many other identifier formats for identifying many other identity document formats. Alternative identifiers for identity documents MUST also be included in the protected header of the COSE envelope. Only one type of identity document identifier MUST be included in a Signed Statement's COSE envelope.
```

Comment:
```suggestion
Transparency Services MAY support other identifier formats for identifying other identity document formats. Alternative identifiers for identity documents MUST also be included in the protected header of the COSE envelope.
```

Reply:
reasoning being that `iss` and `kid` are alternative names for `x5t` so logically, if `iss` and `x5t` are present, we are allowing / requiring multiple identifiers.

Reply:
So including several (potential conflicting identity document identifier is fine? Or do we want text, if there are multiple they MUST relate to the same identity document?

Reply:
If my registration policy accepts your message, I see no reason for us to say more.

The industry pressure for multiple receipts will cause the alignment here, and I doubt we can normatively capture all the possible ways to indirectly refer to an "identity document".

Reply:
Good. I agree. Maybe some Security Considerations? Created #170 as a reminder.